curling/test.py

```diff
@@ -128,8 +131,8 @@ def setUp(self):
         self.api = MockAPI('')
 
     def test_get_one(self):
-        eq_(self.api.services.settings('APPEND_SLASH').get_object(),
-            samples['GET:/services/settings/APPEND_SLASH/'])
+        eq_(self.api.services.setting.get_object(),
+            {'key': 'ABSOLUTE_URL_OVERRIDES'})
```

Comment:
This changes the test a bit, the results `services.settings('APPEND_SLASH')` and `services.setting..` are different and this tests how a result without a meta in the result are handled. If that's covered elsewhere, that's cool.